wiki/Tournaments/en.md

```diff
@@ -326,6 +326,16 @@ Unofficial tournaments/competitions hosted by the communities.
 | :-- | :-- | :-- | :-- | :-- | :-- |
 | osu!mania Asian's Champions 2019 | 2019-09-28 | 2019-11-10 | <!-- TODO: find this flag --> ![][flag___] [\[ R o a d \]](https://osu.ppy.sh/users/6535926) | ![][flag_TW] [\[Crz\]FolAH1217](https://osu.ppy.sh/users/6232458) | ![][flag_MY] [xxxxxx2800](https://osu.ppy.sh/users/4084853) |
 
+### ![](/wiki/shared/mode/osu.png) Royal Danish osu!Cup
```

Comment:
you should put the entire header at the end of osu! tournaments (after `osu! Asia Oceania Tournament`)

Reply:
lately new ones have been going at the bottom, i guess it would make sense to organize by mode/alphabet but lets do that in another pr since we have to update this index as is.